polar-core/src/vm.rs

```diff
@@ -694,7 +694,7 @@ impl PolarVirtualMachine {
         }
     }
 
-    fn source(&self, term: &Term) -> Option<Source> {
+    pub fn source(&self, term: &Term) -> Option<Source> {
```

Comment:
Why does this have to be `pub`?

Reply:
So we can call it from the method on Query.